doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -925,6 +925,19 @@ Other libraries
   * Updated the SUPL client OS integration library to remove the dependency on the newlib C library.
     To use SUPL with picolibc, v0.8.0 or later of the nRF91 Series SUPL client library is required.
 
+* :ref:`emds_readme` library:
+
+  * Added support of the robust data storage mechanism, as it enables the library to always have a valid snapshot of the stored data.
+
+  * Updated:
+
+    * By redesigning the library to support two partitions for storing data.
+    * By redesigning the data format in which the library stores data.
+    * By changing the function :c:func:`emds_store_size_get` for calculation of the needed size to store the registered data.
+    * By changing the function :c:func:`emds_store_time_get` for estimation of the needed time to write the registered data into persistent memory.
+
+  * Removed experimental status.
+
```

Comment:
If this change is for the 3.1.0 release, please remove these changelog entries from this PR and add a comment on PR https://github.com/nrfconnect/sdk-nrf/pull/23631 after this PR is merged.
Also, add the backport label.